p2p/net/swarm/dial_ranker.go

```diff
@@ -112,24 +135,32 @@ func getAddrDelay(addrs []ma.Multiaddr, tcpDelay time.Duration, quicDelay time.D
 
 	res := make([]network.AddrDelay, 0, len(addrs))
 
+	var tcpIdx int
 	var totalTCPDelay time.Duration
 	for i, addr := range addrs {
 		var delay time.Duration
 		switch {
 		case isQUICAddr(addr):
-			// For QUIC addresses we dial an IPv6 address, then after quicDelay an IPv4
-			// address, then after quicDelay we dial rest of the addresses.
```

Comment:
Why was this comment deleted? Does it now apply to both QUIC and TCP? Would be worth keeping it.

Reply:
You're right. I don't remember why I chose to remove this instead of adding a similar comment to the TCP block.